google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -235,6 +248,18 @@ Builder setUniformBucketLevelAccessLockedTime(Long uniformBucketLevelAccessLocke
         return this;
       }
 
+      /**
+       * Sets the bucket's Public Access Prevention configuration. Currently, 'unspecified' and
+       * 'enforced' are supported.
+       *
+       * @see <a
+       *     href="https://cloud.google.com/storage/docs/public-access-prevention">public-access-prevention</a>
+       */
```

Comment:
link does not work
https://cloud.google.com/storage/docs/public-access-prevention

Reply:
@dmitry-fa These URL returns 404s -- is the backend feature not yet public.